### PR TITLE
Add Spacers and Points in zingocli balance command

### PR DESCRIPTION
**Pull request by fluidvanadium**
Adding a decimal point allows easy estimation of zec per zatoshi.

### CODE REVIEW

**Comment by zancas** [comment]
Is this moving us towards reproducing the vaporizing transaction bug?

File: zingolib/src/lightclient.rs [hunk 1]
```diff
@@ -128,6 +128,63 @@ pub struct PoolBalances {
     /// TODO: Add Doc Comment Here!
     pub transparent_balance: Option<u64>,
 }
+fn format_option_zatoshis(ioz: &Option<u64>) -> String {
```
Comment (zancas) [inline zingolib/src/lightclient.rs:131]: I don't know what "`ioz`" means.

**Comment by zancas** [comment]
It looks like this crate has been downloaded aroun 15_000_000 times:

https://crates.io/crates/num-format

Why don't we us it?

**Review by zancas — CHANGES_REQUESTED** [review]
Let's use num-format.

**Comment by zancas** [comment]
I really like the idea of a more useful/readable zingocli interface.   I don't like the idea of writing our own reimplementation of standard logic.   Let's use the thing that's "off-the-shelf".

**Comment by fluidvanadium** [comment]
> It looks like this crate has been downloaded aroun 15_000_000 times:
> 
> https://crates.io/crates/num-format
> 
> Why don't we us it?

the answer is: num-format doesnt specifically make use of the 10,000,000 mark for Satoshis

File: zingolib/src/lightclient.rs (hunk 1 above)
Comment (zancas) [reply zingolib/src/lightclient.rs:131]: Is it like "input_of_zatoshis"?    Could we name this parameter:  "zats"?

**Review by zancas — APPROVED** [review]

**Pull request merged**